packages/node_modules/@ciscospark/internal-plugin-conversation/test/integration/spec/create.js

```diff
@@ -163,6 +179,13 @@ describe('plugin-conversation', function () {
       .then((c) => spark.internal.conversation.get(c, {activitiesLimit: 2}))
       .then((c) => assert.equal(c.activities.items[1].object.displayName, 'comment')));
 
+    it('creates a conversation with a markdown comment', () => spark.internal.conversation.create({comment: 'comment', markdown: '<strong>comment</strong>', participants})
```

Comment:
Similarly, let's use "richtext" or "html" in the test descriptions. Markdown is handled at a higher layer.